src/rdb.c

```diff
@@ -388,18 +388,24 @@ ssize_t rdbSaveLzfBlob(rio *rdb, void *data, size_t compress_len, size_t origina
 ssize_t rdbSaveLzfStringObject(rio *rdb, unsigned char *s, size_t len) {
     size_t comprlen, outlen;
     void *out;
+    static void *buffer = NULL;
```

Comment:
What if we just apply this optimization? This seems like something we might consider doing regardless.

Reply:
This optimization can be its own PR if we prefer to break it down as small as we can

Reply:
My main question is "how much of an affect does this optimization have"?

Reply:
I did a benchmark for this specific change, and it didn't have a measurable effect without the changes in this PR.

Reply:
With the changes in the pr it does?

Reply:
Yes. Without that change the compression-related allocations drive CoW up.